robottelo/decorators/__init__.py

```diff
@@ -325,6 +324,7 @@ def __call__(self, func):
                 config_picker=self.config_picker
         )) or (self.bug_type == 'redmine' and rm_bug_is_open(self.bug_id)):
             func = self._wrapper(func)
+            func.run_in_one_thread = func.pytestmark
```

Comment:
You are adding random property to function object just to make broken test happy. I strongly feel this is not the correct way to solve the issue.

Reply:
Agree, This is not a good practice, I will follow the instruction that you suggested in your previous comment and will update you soon.